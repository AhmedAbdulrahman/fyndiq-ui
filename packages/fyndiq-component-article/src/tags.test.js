import React from 'react'
import { shallow } from 'enzyme'

import Tags from './tags'

describe('Article Tags', () => {
  it('should have a default structure', () => {
    expect(
      shallow(<Tags tags={['tag 1', 'tag 2', 'tag 3']} />),
    ).toMatchSnapshot()
  })

  it('should not render anything if no tags are specified', () => {
    expect(shallow(<Tags />).isEmptyRender()).toBe(true)
  })
})
