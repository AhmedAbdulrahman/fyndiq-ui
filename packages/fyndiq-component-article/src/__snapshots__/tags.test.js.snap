// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article Tags should have a default structure 1`] = `
<div
  className="tagsWrapper"
>
  <span
    className="tag"
    key="tag 1"
  >
    tag 1
  </span>
  <span
    className="tag"
    key="tag 2"
  >
    tag 2
  </span>
  <span
    className="tag"
    key="tag 3"
  >
    tag 3
  </span>
</div>
`;
